tensorflow/python/keras/saving/saving_utils_test.py

```diff
@@ -205,5 +207,38 @@ def test_model_save(self):
         {model.output_names[0]: model.predict_on_batch(inputs)},
         _import_and_infer(save_dir, {model.input_names[0]: np.ones((8, 5))}))
 
+
+class ExtractModelMetricsTest(test.TestCase):
+
+  def test_extract_model_metrics(self):
+    a = keras.layers.Input(shape=(3,), name='input_a')
+    b = keras.layers.Input(shape=(3,), name='input_b')
+
+    dense = keras.layers.Dense(4, name='dense')
+    c = dense(a)
+    d = dense(b)
+    e = keras.layers.Dropout(0.5, name='dropout')(c)
+
+    model = keras.models.Model([a, b], [d, e])
+    optimizer = RMSPropOptimizer(learning_rate=0.001)
```

Comment:
nit: change `optimizer = 'rmsprop'`, which allows Keras to map this to the correct optimizer